azure/azure-explanation/security-overview.rst

```diff
@@ -0,0 +1,109 @@
+Ubuntu also produces images which support `Azure Confidential Virtual Machines <https://learn.microsoft.com/en-us/azure/confidential-computing/confidential-vm-overview>`_.
+
+See :doc:`confidential-computing`.
+
+Choosing the right image
+------------------------
+
+Ubuntu offers a range of image types on Azure to meet security requirements. The security features and capabilities
+of each image can be further enhanced with `Ubuntu Pro`_.
+
+Server
+~~~~~~
+
+Optimised for general server use cases, Ubuntu Server is ideal for users seeking a stable and secure OS foundation to
+support server workloads.
+
+Minimal
+~~~~~~~
+
+By providing a greatly reduced package set to minimise the security cross-section of the image, Ubuntu Minimal requires
+fewer security updates over time.
+
+CIS
+~~~
+
+A hardened Ubuntu image (see `Image Hardening`_), pre-configured to align with industry best practices and guidelines
+set by the Center for Internet Security (CIS). Ubuntu CIS images on Azure use custom tailoring files to meet both CIS
+benchmarks as well as ensure the image remains functional in Azure cloud environments.
+
+FIPS
+~~~~
+
+The Ubuntu FIPS image contains a Ubuntu kernel and security components which have been certified by the National
+Institute of Standards and Technology (NIST), making it a compelling choice for users operating in regulated
+environments such as FedRAMP, HIPAA, or PCI.
+
+Confidential VM
+~~~~~~~~~~~~~~~
+
+Supports Azure Confidential Virtual Machines, providing enhanced security features to protect data at rest, in use, and
+during boot.
+
+Ubuntu Pro
+----------
+
+Ubuntu Pro extends the capabilities of Ubuntu Server by incorporating advanced security, compliance features, and
+systems management tooling. It is designed for production environments where additional security layers and
+compliance with regulatory standards are paramount. Ubuntu Pro includes:
+
+* Expanded Security Maintenance: Security updates and patches for a wide array of applications and packages beyond the
+  standard offerings.
+* Kernel Livepatch: Minimises downtime and disruption by providing kernel updates that can be applied without rebooting
+  the system.
+* Up to 12 years of support: Long-term support that ensures your systems are secure and stable for more than a decade,
+  with a guaranteed upgrade path.
+
+To learn more about how to get Ubuntu Pro, see :doc:`../azure-how-to/instances/get-ubuntu-pro`.
+
+Image Hardening
+---------------
+
+The security profile for Ubuntu images on Azure can be further enhanced with `hardening <https://ubuntu.com/blog/what-is-system-hardening-definition-and-best-practices>`_
+by applying CIS hardening profiles from the `Ubuntu Security Guide (USG) <https://ubuntu.com/security/certifications/docs/usg>`_.
+
+Updates
+-------
+
+Latest Images
+~~~~~~~~~~~~~
+
+Ubuntu images for a given Azure Marketplace offer (see :doc:`../azure-how-to/instances/find-ubuntu-images`) are
+regularly published to deliver:
```

Comment:
Do we have a documentation page/blog post explaining our release cadence? (every time there is a change to the package release set and what is this package release set)

Reply:
It is briefly covered in the `Understanding Ubuntu on Azure` explanation document, but that description is also limited to the kernel (no mention of the release package set). My question is if we want to even expose users reading public documentation to the concept of the release package set (is this something that is even advertised anywhere else. Will customers complain that X Y Z packages are not part of the package set, etc.).

Reply:
> Will customers complain that X Y Z packages are not part of the package set

Maybe, but our rule for including packages in this set is very clear: anything that would require a reboot to be fully updated. So it doesn't allow for a lot of debates.